plugins/watch-pr/skills/watch-pr/watch-pr.py

```
@@ -345,22 +406,26 @@ def emit(line: str) -> None:
         review_comments = [item for item in json_array("gh", "api", "--paginate", f"repos/{slug}/pulls/{number}/comments") if (item.get("user") or {}).get("login") != me]
+        current_check_states = check_states(checks)
         current = set(state_lines(meta, checks, slug, origin_matches, me, len(review_comments), comment_reactions(slug, number, me)))
         added = current - previous
         unresolved_delta = unresolved_delta_line(previous_unresolved, unresolved)
         unresolved_added = unresolved - previous_unresolved
         if unresolved_added or any(line.startswith(("review ", "comments: ", "review-comments: ")) for line in added):
             # Fresh feedback: owe a fetch, and hand the formatter a clean retry budget.
             pending_fetch, failures = True, 0
-        for line in sorted(added): emit(line)
+        check_lines = {line for line in added if line.startswith("check ")}
+        for line in sorted(added - check_lines): emit(line)
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)